contrib/ay-openqa-worker-leap-3-nvme.xml

```diff
@@ -0,0 +1,150 @@
+    </routing>
+  </networking>
+  <add-on>
+    <add_on_others t="list">
+      <listentry>
+        <media_url>https://download.opensuse.org/distribution/leap/$releasever/repo/oss</media_url>
+        <alias>repo-oss</alias>
+        <name>repo-oss</name>
+      </listentry>
+      <listentry>
+        <media_url>https://download.opensuse.org/distribution/leap/$releasever/repo/non-oss</media_url>
+        <alias>repo-non-oss</alias>
+        <name>repo-non-oss</name>
+      </listentry>
+      <listentry>
+        <media_url>https://download.opensuse.org/update/leap/$releasever/oss</media_url>
+        <alias>repo-update-oss</alias>
+        <name>repo-update-oss</name>
+      </listentry>
+      <listentry>
+        <media_url>https://download.opensuse.org/update/leap/$releasever/non-oss</media_url>
+        <alias>repo-update-non-oss</alias>
+        <name>repo-update-non-oss</name>
+      </listentry>
+    </add_on_others>
+  </add-on>
+  <software>
+    <products t="list">
+      <product>Leap</product>
+    </products>
+    <patterns t="list">
+      <pattern>kvm_server</pattern>
+    </patterns>
+    <packages t="list">
+      <package>openssh</package>
+      <package>sudo</package>
+    </packages>
+  </software>
+  <timezone t="map">
+    <hwclock>UTC</hwclock>
+    <timezone>Etc/UTC</timezone>
+  </timezone>
+  <partitioning t="list">
+    <drive>
+      <type t="symbol">CT_DISK</type>
+      <device>/dev/nvme1n1</device>
+      <disklabel>none</disklabel>
+      <use>all</use>
+      <partitions t="list">
+        <partition>
+          <raid_name>/dev/md0</raid_name>
+        </partition>
+      </partitions>
+    </drive>
+    <drive>
+      <type t="symbol">CT_DISK</type>
+      <device>/dev/nvme2n1</device>
+      <disklabel>none</disklabel>
+      <use>all</use>
+      <partitions t="list">
+        <partition>
+          <raid_name>/dev/md0</raid_name>
+        </partition>
+      </partitions>
+    </drive>
+    <drive>
+      <type t="symbol">CT_MD</type>
+      <device>/dev/md0</device>
+      <disklabel>gpt</disklabel>
+      <use>all</use>
+      <raid_options>
+        <raid_type>raid1</raid_type>
+        <device_order t="list">
+          <device>/dev/nvme1n1</device>
+          <device>/dev/nvme2n1</device>
+        </device_order>
+      </raid_options>
+      <partitions t="list">
+        <partition>
+          <mount>/boot/efi</mount>
+          <size>512MiB</size>
+          <filesystem t="symbol">vfat</filesystem>
+        </partition>
+        <partition>
+          <mount>/</mount>
+          <size>max</size>
+          <filesystem t="symbol">btrfs</filesystem>
+        </partition>
+      </partitions>
+    </drive>
+    <drive>
+      <type t="symbol">CT_DISK</type>
+      <device>/dev/nvme0n1</device>
+      <disklabel>gpt</disklabel>
+      <use>all</use>
+      <partitions t="list">
+        <partition>
+          <size>max</size>
+          <filesystem t="symbol">ext4</filesystem>
```

Comment:
iont this reformatted by service to ext2 ?

Reply:
only if we actually use that service which is AFAIK only the case in OSD with salt

Reply:
I think ext4 is a good default. If it gets overridden that's not a problem.